js/languagePickerView.js

```diff
@@ -22,7 +22,8 @@ define([
             var data = this.model.toJSON();
             var template = Handlebars.templates[this.constructor.template];
             this.$el.html(template(data));
```

Comment:
I think it is a bit cleaner, but I am more than happy to go with the current approach.